utils/travis/publish-gh-pages.sh

```diff
@@ -2,5 +2,5 @@
 set -x
 set -e
 
-yarn build:demo
+yarn build:demo_changed
```

Comment:
Just use `yarn build:demo --since` here. (Wouldn't `--since` be a better choice than `--since HEAD^`?)

Reply:
By default it will run against anything since the last tag https://github.com/lerna/lerna#--since-ref, which should be fine!

We originally had this in the publish script, but we changed it to `HEAD^`. Not sure what the reasoning was for that though.

Fixed.

Reply:
On the publish side, we need to prevent inadvertent publish attempts on packages that may have been excluded (for one reason or another) from the `yarn run tag` version updates. In this case, we want to be more aggressive about picking everything up for the demo build.